frontend/src/components/ViewTrips/manage-trips-bar.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { Button, Row, Col, Nav } from 'react-bootstrap';
+
+import TripView from '../../constants/trip-view.js';
+
+/**
+ * Component that contains the tabs to switch the trip view as well as the
+ * add new trip button.
+ *
+ * @property {Object} props These are the props for this component:
+ * @property {Function} props.handleAddTrip Event handler responsible for
+ *     displaying the add trip modal.
+ * @property {Function} props.handleChangeView Event handler responsible for
+ *     updated the `tripView` state in `ViewTrips/index.js`.
+ * @property {string} props.tripView The current user's trips page view.
+ */
+const ManageTripsBar = (props) => {
+  const active = TripView.ACTIVE;
+  const pending = TripView.PENDING;
+  const rejected = TripView.REJECTED;
+
+  return (
+    <Row>
+      <Col md={10} sm={9} xs={8}>
+        <Nav
+          variant="tabs"
+          activeKey={props.tripView}
+          defaultActiveKey={active}
+        >
+          <Nav.Item>
+            <Nav.Link
+              eventKey={active}
+              onSelect={() => props.handleChangeView(active)}
+            >
+              Active
+            </Nav.Link>
+          </Nav.Item>
```

Comment:
Make a function to generate each of these? Up to you.

Reply:
Yeah that would look a lot cleaner, I will add that. Thanks for the suggestion!